build.gradle

```diff
@@ -133,6 +133,10 @@ dependencyCheck {
   }
 }
 
+def versions = [
+  lombok          : '1.18.24'
+]
```

Comment:
Any particular reason why the lombok version has been declared this way rather than adding it to the ext block after it?

Reply:
Confirmed that declaration has been moved to ext block.